test/sparse/higherorderfns.jl

```diff
@@ -76,20 +75,77 @@ end
     end
 end
 
-@testset "broadcast[!] implementation specialized for a single (input) sparse vector/matrix" begin
-    # broadcast[!] for a single sparse vector/matrix falls back to map[!], tested extensively
-    # above. here we simply lightly exercise the relevant broadcast[!] entry points.
+@testset "broadcast implementation specialized for a single (input) sparse vector/matrix" begin
+    # broadcast for a single (input) sparse vector/matrix falls back to map, tested
+    # extensively above. hre we simply lightly exercise the relevant broadcast entry
+    # point.
     N, M, p = 10, 12, 0.4
     a, A = sprand(N, p), sprand(N, M, p)
     fa, fA = Array(a), Array(A)
     @test broadcast(sin, a) == sparse(broadcast(sin, fa))
     @test broadcast(sin, A) == sparse(broadcast(sin, fA))
-    @test broadcast!(sin, copy(a), a) == sparse(broadcast!(sin, copy(fa), fa))
-    @test broadcast!(sin, copy(A), A) == sparse(broadcast!(sin, copy(fA), fA))
+    # The two following tests are from #19895, and check the absence of an ambiguity
+    # between a pair of specializations, but #19895 removed both of those specializations,
+    # so perhaps these tests should be nixed?
```

Comment:
A few months down the road, someone decides to add specializations for the common  `x .= y` case to be faster...